x/ibc/04-channel/types/msgs_test.go

```diff
@@ -416,7 +416,7 @@ func (suite *MsgTestSuite) TestMsgRecvPacketGetSignBytes() {
 	res := msg.GetSignBytes()
 
 	expected := fmt.Sprintf(
-		`{"packet":{"data":%s,"destination_channel":"testcpchannel","destination_port":"testcpport","sequence":"1","source_channel":"testchannel","source_port":"testportid","timeout_height":"100","timeout_timestamp":"100"},"proof":"Co0BCi4KCmljczIzOmlhdmwSA0tFWRobChkKA0tFWRIFVkFMVUUaCwgBGAEgASoDAAICClsKDGljczIzOnNpbXBsZRIMaWF2bFN0b3JlS2V5Gj0KOwoMaWF2bFN0b3JlS2V5EiAcIiDXSHQRSvh/Wa07MYpTK0B4XtbaXtzxBED76xk0WhoJCAEYASABKgEA","proof_height":"1","signer":"cosmos1w3jhxarpv3j8yvg4ufs4x"}`,
```

Comment:
hmmm, this needs to be fixed?

Reply:
don't think so? this is the result of switching from amino json encoding to proto json encoding

Reply:
the proto json should contain the underscores too afaik? cc @aaronc @clevinson

Reply:
this was in the channel.pb.go file ``protobuf:"bytes,2,opt,name=source_port,json=sourcePort,proto3" json:"source_port,omitempty" yaml:"source_port"``

is there a tag we can add to the .proto files to set the json tag?

Reply:
or maybe this should be just added to `make proto-gen`

Reply:
can you open an issue? 🙏